languages/elixir/exercises/concept/binary-matching/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+You have been working on a project which allows users to upload files to the server to be shared with other users. You have been tasked write a function to verify that an upload matches it's [media type][mimetype]. You do some research and discover that the first few bytes of a file is generally unique to that filetype, giving it a sort of signature.
+
+Use the following table for reference:
+
+| File type | Common extension  | Media type                   | binary 'signature'                               |
+| --------- | ----------------- | ---------------------------- | ------------------------------------------------ |
+| ELF       | `none` or `"exe"` | `"application/octet-stream"` | `0x7F, 0x45, 0x4C, 0x46`                         |
```

Comment:
When I read the table, I thought I also need to handle a "no extension" case in `type_from_extension`. I see now that that is not required, so maybe the table doesn't need to mention that ELF files can have no extension?

Reply:
```suggestion
| ELF       | `"exe"` | `"application/octet-stream"` | `0x7F, 0x45, 0x4C, 0x46`                         |
```

Reply:
This commit will cause a formatting error, which I will resolve in a follow up commit.